### PR TITLE
add recipes for freelancers

### PR DESCRIPTION
**Pull request by sakarimov**
- **add recipes for freelancers**
- **edit recipes for freelancers after test**

#### Pre-flight Checklist

Please ensure you've completed all of the following.

- [x] I have read the [Contributing Guidelines](https://github.com/ferdium/ferdium-app/blob/HEAD/CONTRIBUTING.md) for this project.
- [x] I agree to follow the [Code of Conduct](https://github.com/ferdium/ferdium-app/blob/HEAD/CODE_OF_CONDUCT.md) that this project adheres to.

## Adding new recipe

Service: services for freelancers, it contains various freelancers platform, i intended to add more of if, but for now i already have 3 of those

URL: guru.com, truelancer.com, peopleperhour.com

Service ID: guru.com, truelancer, peopleperhour

### Terminal output

```bash
> ferdium-recipes@1.0.1 package /run/media/officestorage/karya/contrib/github/ferdium-recipes
> node scripts/package.js

✅ Successfully packaged and added 348 recipes (0 unsuccessful recipes)
```

### Additional information

- [x] I am the original creator of this package
- [x] I have run the `pnpm lint:fix && pnpm reformat-files && pnpm package` and verified that there are no validation errors reported for this package
- [x] My recipe has been tested to work inside Ferdium

### CODE REVIEW

File: .npmrc [hunk 1]
```diff
@@ -4,3 +4,4 @@ unsafe-perm = true
 frozen-lockfile = false
 legacy-peer-deps = true
 shamefully-hoist = true
+package-manager-strict=false
```
Comment (SpecialAro) [inline .npmrc:7]: Why is this needed?

File: recipes/peopleperhour/package.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+  "id": "peopleperhour",
+  "name": "People Per Hour",
+  "version": "0.1.1",
```
Comment (SpecialAro) [inline recipes/peopleperhour/package.json:4]: Can we please start on `1.0.0` ?

File: recipes/guru/package.json [hunk 3]
```diff
@@ -0,0 +1,9 @@
+{
+  "id": "guru",
+  "name": "guru.com",
+  "version": "0.1.1",
```
Comment (SpecialAro) [inline recipes/guru/package.json:4]: Can we please start on `1.0.0` ?

File: recipes/truelancer/package.json [hunk 4]
```diff
@@ -0,0 +1,9 @@
+{
+  "id": "truelancer",
+  "name": "Truelancer",
+  "version": "0.1.1",
```
Comment (SpecialAro) [inline recipes/truelancer/package.json:4]: Can we please start on `1.0.0` ?

File: recipes/truelancer/icon.svg
Comment (SpecialAro) [inline recipes/truelancer/icon.svg:1]: Can we only use the logo itself and not the words?

File: recipes/guru/icon.svg
Comment (SpecialAro) [inline recipes/guru/icon.svg:1]: Can we use the logo itself and not the words?

File: recipes/peopleperhour/icon.svg
Comment (SpecialAro) [inline recipes/peopleperhour/icon.svg:1]: Can we use the logo itself and not the words?

**Review by SpecialAro — COMMENTED** [review]
Hi @sakarimov, thank you for your contribution.

Please fix my review comments whenever you can 🙏

Thank you!

File: .npmrc (hunk 1 above)
Comment (sakarimov) [reply .npmrc:7]: sorry, i forgot to remove that line, while project preparation i was stuck with version issue, i will remove that line soon

File: recipes/peopleperhour/package.json (hunk 2 above)
Comment (sakarimov) [reply recipes/peopleperhour/package.json:4]: okay will do it

File: recipes/guru/package.json (hunk 3 above)
Comment (sakarimov) [reply recipes/guru/package.json:4]: okay will do it

File: recipes/truelancer/package.json (hunk 4 above)
Comment (sakarimov) [reply recipes/truelancer/package.json:4]: okay will do it

File: recipes/truelancer/icon.svg
Comment (sakarimov) [reply recipes/truelancer/icon.svg:1]: this is the actual logo, but i will remove it, since it's better without the text

File: recipes/guru/icon.svg
Comment (sakarimov) [reply recipes/guru/icon.svg:1]: this is the actual logo, but i will remove it, since it's better without the text

File: recipes/peopleperhour/icon.svg
Comment (sakarimov) [reply recipes/peopleperhour/icon.svg:1]: this is the actual logo, but i will remove it, since it's better without the text

**Comment by sakarimov** [comment]
everything resolved, thank u for the guidance @SpecialAro

File: recipes/peopleperhour/webview.js [hunk 5]
```diff
@@ -0,0 +1,13 @@
+//const path = require('path');
```
Comment (vraravam) [inline recipes/peopleperhour/webview.js:1]: please resurrect the support for injecting the `service.css` which should have been auto-generated when the code was generated from the template.

File: recipes/guru/webview.js [hunk 6]
```diff
@@ -0,0 +1,13 @@
+//const path = require('path');
+
```
Comment (vraravam) [inline recipes/guru/webview.js:2]: please resurrect the support for injecting the `service.css` which should have been auto-generated when the code was generated from the template.

File: recipes/truelancer/webview.js [hunk 7]
```diff
@@ -0,0 +1,13 @@
+//const path = require('path');
+
```
Comment (vraravam) [inline recipes/truelancer/webview.js:2]: please resurrect the support for injecting the `service.css` which should have been auto-generated when the code was generated from the template.

**Review by vraravam — CHANGES_REQUESTED** [review]

File: recipes/guru/webview.js (hunk 6 above)
Comment (sakarimov) [reply recipes/guru/webview.js:6]: i have resurrected the support for service.css, i adopted the code from another recipe, let me know if it's not working

File: recipes/truelancer/webview.js (hunk 7 above)
Comment (sakarimov) [reply recipes/truelancer/webview.js:6]: i have resurrected the support for service.css, i adopted the code from another recipe, let me know if it's not working

File: recipes/peopleperhour/webview.js (hunk 5 above)
Comment (sakarimov) [reply recipes/peopleperhour/webview.js]: i have resurrected the support for service.css, i adopted the code from another recipe, let me know if it's not working

**Comment by sakarimov** [comment]
i want to resolve issue on ferdium/ferdium-app#1742 to add recipe of upwork should i create new pr or can i just add the recipe here? @vraravam @SpecialAro

**Comment by vraravam** [comment]
> i want to resolve issue on [ferdium/ferdium-app#1742](https://github.com/ferdium/ferdium-app/issues/1742) to add recipe of upwork should i create new pr or can i just add the recipe here? @vraravam @SpecialAro

pls create a new PR

**Comment by vraravam** [comment]
@all-contributors please add @sakarimov for code
